examples/write_log_events.rs

```diff
@@ -194,9 +194,9 @@ mod example {
                 .write(11, event.thread_id)?
                 .write(12, &event.thread_name)?
                 //                 v--- The fixed strings from the log statement are also SIDs
-                .write(13, RawSymbolToken::SymbolId(17))? // logger name
-                .write(14, RawSymbolToken::SymbolId(18))? // log level
-                .write(15, RawSymbolToken::SymbolId(19))? // format
+                .write(13, RawSymbolTokenRef::SymbolId(17))? // logger name
+                .write(14, RawSymbolTokenRef::SymbolId(18))? // log level
+                .write(15, RawSymbolTokenRef::SymbolId(19))? // format
```

Comment:
🗺️ Now that `RawSymbolTokenRef` can hold a reference to a decoded string in the bump allocator, there's no longer a need for a separate `RawSymbolToken` type that can own a string.

Down the line, I'd like to rename `RawSymbolTokenRef` to simply `RawSymbol`.